introduce php56 version

bparees opened the pull request:
based on work originally done by @remicollet here: https://github.com/openshift/sti-php/pull/42

bparees commented:
@mfojtik ptal again.

mfojtik commented on 5.6/.sti/bin/assemble:
```diff
@@ -0,0 +1,20 @@
+cp -Rf /tmp/src/. ./
+
+if [ -f composer.json ]; then
+  echo "Installing dependencies from composer.json... "
```
Can you  make this message consistent with above? `" ..."`.
Also I think it would make more sense to change this to something like:

`Found 'composer.json', installing dependencies using composer.phar ...`

(so users know what is going on ;)

bparees replied:
will fix here and in 5.5

mfojtik commented on 5.6/.sti/bin/usage:
```diff
@@ -0,0 +1,18 @@
+
+cat <<EOF
+This is a STI PHP-5.6 ${DISTRO} base image:
+To use it, install STI: https://github.com/openshift/source-to-image
```
s/STI/S2I/g

bparees replied:
will fix here and in 5.5

mfojtik commented on 5.6/Dockerfile:
```diff
@@ -0,0 +1,52 @@
+    yum install -y --setopt=tsflags=nodocs httpd24 \
+    rh-php56 rh-php56-php rh-php56-php-mysqlnd rh-php56-php-pgsql rh-php56-php-bcmath \
+    rh-php56-php-gd rh-php56-php-intl rh-php56-php-ldap rh-php56-php-mbstring rh-php56-php-pdo \
+    rh-php56-php-pecl-memcache rh-php56-php-process rh-php56-php-soap rh-php56-php-opcache rh-php56-php-xml \
```
I wish there is a virtual RPM that installs all the packages needed...

mfojtik commented on 5.6/Dockerfile.rhel7:
```diff
@@ -0,0 +1,62 @@
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,php,php56,rh-php56"
+
+LABEL k8s.io/description="Platform for building and running PHP 5.6 applications" \
```
Do we care about these labels?

bparees replied:
we do not..will remove here and in 5.5

mfojtik commented on 5.6/test/run:
```diff
@@ -0,0 +1,208 @@
+    echo "ERROR[/bin/bash -c "${run_cmd}"] Expected '${expected}', got '${out}'"
+    return 1
+  fi
+  # out=$(docker run --rm -i ${IMAGE_NAME} /bin/bash -i <<< "${run_cmd}")
```
Can you nuke this?

bparees replied:
nuked here and in 5.5

mfojtik commented:
@bparees LGTM (few small nits)

bparees commented:
@mfojtik i think i addressed all the comments...currently failing because software collections is down, so i can't get far enough to see if there are any other issues.

bparees commented:
[test]

mfojtik commented:
LGTM

bparees commented:
[test]

bparees commented:
[test]
